### DISCUSSION TITLE
Is already any 8bit-font available?

### DISCUSSION THREAD

**Discussion by brutzler** [lovyan03/LovyanGFX#602] [General]
I need for displaying some temperatures the °-sign.
As it is not part of any 7bit-font, I was looking for a 8 bit font within the lib. 
But did not found any :-(
Will this be supported in future, or is there any workarond?

**Comment by WeLizard** [comment]
I was very disappointed with this library, I couldn't even output .vlw fonts on 8-bit parallel display
